###################  Command 
string command      


---

bool result

# Result of Relocation
string[] action_sequence
